src/core/snappingutils.cpp

```diff
@@ -20,7 +20,7 @@
 #include "qgsproject.h"
 
 SnappingUtils::SnappingUtils( QObject *parent )
-  : QgsSnappingUtils( parent )
+  : QgsSnappingUtils( parent, false /*enableSnappingForInvisibleFeature*/ )
```

Comment:
```suggestion
  : QgsSnappingUtils( parent, false )
```
Is this comment only to remember, that we could have a setting for that in future?
If so, then I would remove it, and maybe open an enhancement-request for it's implementation instead.
But up to you.

Reply:
So no removing :slightly_smiling_face:

Reply:
it's to know what this is used for without having to look in the API docs.
or why using flags instead of booleans.

Reply:
Okay. It's fine.